src/components/hooks/useTabState.ts

```diff
@@ -1,7 +1,7 @@
 import { useState } from 'react';
 
 export function useTabState(
-    tabs: { [k: string]: string },
+    tabs: { [k: string]: string | object },
```

Comment:
Should we type this?

```suggestion
    tabs: { [k: string]: string | { id: string; label: string } },
```

Reply:
Good suggestion. At the moment, the shape of the object doesn't matter. So I'm okay leaving it generic.